app/services/analytics.rb

```diff
@@ -55,6 +55,7 @@ def browser
 
   ACCOUNT_DELETION = 'Account Deletion Requested'.freeze
   ACCOUNT_VISIT = 'Account Page Visited'.freeze
+  ADDRESS_DELIVERY_METHOD = 'IdV: Address Verification Selection'.freeze
```

Comment:
Thanks for updating the string text. Could we also please update the constant name to `IDV_ADDRESS_VERIFICATION_SELECTION`? I think that better describes what the user is doing. Thoughts?